index.bs

```diff
@@ -584,7 +632,12 @@ and an {{IntersectionObserverInit}} dictionary |options|, run these steps:
 8. If |thresholds| is empty, append <code>0</code> to |thresholds|.
 9. The {{IntersectionObserver/thresholds}} attribute getter will return
 	this sorted |thresholds| list.
-10. Return |this|.
+10. If |options|.{{IntersectionObserverInit/trackVisibility}} is <code>true</code>,
+	and |options|.{{IntersectionObserverInit/delay}} is not specified or is less than <code>100</code>,
+	<a>throw</a> a {{NotAllowedError}} exception.
+11. Set |this|'s internal {{[[delay]]}} slot to |options|.{{IntersectionObserverInit/delay}}.
```

Comment:
Do you need to set this if `trackVisibility` is `false`?

Reply:
There was prior working group  discussion about the `delay` parameter being useful even when `trackVisibility` is false (sorry I can't find a record of this, I think it was a working group call from early 2023). I can split the `delay` feature into a separate PR to land before this one, if you prefer.

Reply:
That's a good enough answer for me, I just noted it as a possibly unused value in case it was a mistake.